### PR TITLE
Comment out the JQuery loader

### PR DESCRIPTION
**Pull request by benjiebeetle**

### CODE REVIEW

File: js/app/cuttingTheMustard.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ Modernizr.load({
 });
 //set conditional assets for main.js
 var globalSiteSpecificVars = {
-	pathToJquery: '//cdn.ucl.ac.uk/indigo/js/lib/jquery-1.9.1.min'
-	,googleAnalyticsIdsArray: []//specify array of site specific id's NOT UCL generic UA-943297-1
+//	pathToJquery: '//cdn.ucl.ac.uk/indigo/js/lib/jquery-1.9.1.min',
```
Comment (gideoncresswell) [inline js/app/cuttingTheMustard.js]: Main.js has `jquery: globalSiteSpecificVars.pathToJquery`. Which sounds to me like it's expecting PathToQuery to have a value. Although where that is then used I don't know...!
e.g. `https://cdn.ucl.ac.uk/skins/UCLDrupalIndigoSkin/default-theme/main.js`

Comment (benjiebeetle) [reply js/app/cuttingTheMustard.js]: I'll have a look...

Comment (benjiebeetle) [reply js/app/cuttingTheMustard.js]: New version to look at using the Drupal JS version.

**Comment by benjiebeetle** [comment]
Close for now, not needed. Other ways forward look more useful.

**Pull request closed without merging**